folium/features.py

```diff
@@ -241,6 +241,14 @@ def __init__(self, data, style_function=None):
             else:  # This is a filename
                 self.embed = False
                 self.data = data
+        elif hasattr(data, 'to_crs'):
```

Comment:
Don't know what would be the best approach :expressionless: 

Sometimes I think that empowering users by showing how to convert a `GeoDataFrame` to `GeoJSON`, and its pyproj like crs to EPSG would be better than wrapping all that in folium...